run_tests.py

```diff
@@ -7,7 +7,7 @@
 import argparse
 
 
-def run_python_tests(coverage=False):
+def run_python_tests(coverage=True):
```

Comment:
why are you changing this?

Reply:
I was testing some thing. Will change it back and commit